res/TensorFlowLiteRecipes/Where_000/test.recipe

```diff
@@ -1,33 +1,19 @@
 operand {
   name: "ifm"
   type: BOOL
-  shape { dim: 4 dim: 1 dim: 1 dim: 1 }
-}
-operand {
-  name: "x"
-  type: FLOAT32
-  shape { dim: 4 dim: 4 dim: 4 dim: 3 }
-}
-operand {
-  name: "y"
-  type: FLOAT32
-  shape { dim: 4 dim: 4 dim: 4 dim: 3 }
+  shape { dim: 4 dim: 2 dim: 3}
 }
 operand {
   name: "ofm"
-  type: FLOAT32
-  shape { dim: 4 dim: 4 dim: 4 dim: 3 }
+  type: INT64
+  shape { } # output shape is 2d, but partially unknown, because first dimension depends on number of "true" values in input so in our case it is (?, 3)
```

Comment:
Can you please split this comment so that length would be in 80 or 100 columns ? Just for better readability :)

Reply:
Fixed, thank you.

P.S. I also changed comment in second recipe to "shape is intentionally omitted", because I found that is the comment used in other recipes =)